### DISCUSSION TITLE
Does this work with Windows 7?

### DISCUSSION THREAD

**Discussion by YumSoup** [kraxarn/spotify-qt#180] [Q&A] [answered]
I can't find anything saying that it doesn't, or that it's exclusive to win10. However, I can't get it to work on windows 7.
This error is encountered on launch:

> Entry Point Not Found : The procedure entry point CreateDXGIFactory2 could not be located in the dynamic link library dxgi.dll

[This forum post](https://forum.qt.io/topic/78380/entry-point-not-found-the-procedure-entry-point-createdxgifactory2-could-not-be-located-in-the-dynamic-link-library-dxgi-dll) on the same error in QT is the closest I could find to suggest SpotifyQT doesn't work for Win7 (and it isn't the fault of my system in particular.

I didn't want to submit an issue as I'm not sure if it is intended to work for win7 or not.

**Comment by kraxarn** [comment, marked as answer]
The 64-bit builds use Qt 6, which require Windows 10 or later. However, the 32-bit builds use Qt 5, which should support Windows 7.
Did you try the 32-bit builds?
